torch/nn/quantized/modules/conv.py

```diff
@@ -15,6 +15,14 @@
 
 
 class Conv2d(_ConvNd):
+    r"""A quantized Conv2d module.
+
+    We adopt the same interface as :class:`torch.nn.Conv2d`.
+    """
+
+    # __QAT_MODULE = QATConv2d
```

Comment:
Removed this? As this file is for the post training quantization instead of quantization aware training.

Reply:
oh, this is a mapping from qat module, right now it has some dependency issue so we can't import qat modules.

Reply:
I'll fix the dependency issue later